pilota-build/test_data/thrift_workspace_with_split/output/article/src/message_Author.rs

```diff
@@ -0,0 +1 @@
+pub use ::common::author::Author;
```

Comment:
why do we need to `pub use` here instead of only `use`? @Millione

Reply:
Users can avoid importing the common crate dependencies